tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml

```diff
@@ -221,6 +221,9 @@ apps:
   shutdown:
     command: bin/run
     plugs: [ shutdown ]
+  snap-prompting-control:
```

Comment:
Maybe a simple standalone spread test would give better coverage? something like `tests/main/interfaces-snap-refresh-observe` .

Note, the spread test checks were updated that if a test named `interfaces-snap-prompting-control` exists it wouldn't complain about it being absent from `test-snapd-policy-app-consumer`.

This can be done in a follow up PR, it is not urgent.

Reply:
Great point, thank you! I'll add this now, since we're blocked by GH actions at the moment anyway